Dockerfile

```diff
@@ -115,6 +115,9 @@ COPY --link --from=buildctl /usr/bin/buildctl /
 FROM scratch AS binaries-windows
 COPY --link --from=buildctl /usr/bin/buildctl /buildctl.exe
 
+FROM scratch AS binaries-freebsd
+COPY --from=buildctl /usr/bin/buildctl /
```

Comment:
FreeBSD only has the `buildctl` CLI, and doesn't compile the `buildkitd`, or does it do _both_ ?

If it _does_ do both, perhaps we should change the `binaries-linux`  stage to `binaries-unix` and add aliases for `-linux` and `-freebsd` (although there's a `binaries-linux-helper` stage as well; I haven't looked closely if that applies to FreeBSD?); my thinking was something like;

```dockerfile
FROM binaries-unix-helper AS binaries-unix
FROM binaries-unix AS binaries-linux
FROM binaries-unix AS binaries-freebsd
```

Reply:
Aha, `binaries-linux-helper` seems to bundle `runc` which is linux-only.